include/channel_table.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * @file channel_table.h
+ * @brief Pr
+ */
+
+#ifndef INCLUDE_CHANNEL_TABLE_H_
+#define INCLUDE_CHANNEL_TABLE_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "charbuf.h"
+
+typedef struct ChannalKeyTable
```

Comment:
Seems like a typo, even though this name isn't used elsewhere.

```suggestion
typedef struct ChannelKeyTable
```